src/number_theory/zsqrtd/basic.lean

```diff
@@ -221,6 +225,65 @@ begin
     exact ⟨hr, hi⟩ },
 end
 
+@[simp, norm_cast]
+lemma coe_int_dvd_coe_int {d : ℤ} (a b : ℤ) : (a : ℤ√d) ∣ b ↔ a ∣ b :=
```

Comment:
Isn't there a global `{d : ℤ}` floating around, meaning you don't need to repeat it on each lemma?